crates/ironrdp-server/src/display.rs

```diff
@@ -84,17 +84,17 @@ impl TryInto<Framebuffer> for BitmapUpdate {
 impl Framebuffer {
     pub fn new(width: NonZeroU16, height: NonZeroU16, format: PixelFormat) -> Self {
         let mut data = BytesMut::new();
-        let w = usize::from(width.get());
-        let h = usize::from(height.get());
+        let w = NonZeroUsize::from(width);
+        let h = NonZeroUsize::from(height);
         let bpp = usize::from(format.bytes_per_pixel());
-        data.resize(bpp * w * h, 0);
+        data.resize(bpp * w.get() * h.get(), 0);
```

Comment:
suggestion: I think it’s best to use the `.get()` only once, instead of everywhere, just to remove noise

```rust
        let w = NonZeroUsize::from(width).get();
        let h = NonZeroUsize::from(height).get();
        let bpp = usize::from(format.bytes_per_pixel());
        data.resize(bpp * w * h, 0);
        data.resize(bpp * w * h, 0);
```

In this case, I don’t think it changes anything, but it’s fine.